cds/compiler/gcc/x86/ts_hardwaretimestamp.h

```diff
@@ -22,6 +23,15 @@ namespace cds { namespace tshardwaretimestamp {
             return ret;
         }
 
+        static inline int has_rdtscp()
```

Comment:
It's quit strange to have "has_..." function returning logically boolean value with int signature